cvmfs/swissknife_lease_json.cc

```diff
@@ -0,0 +1,94 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "swissknife_lease_json.h"
+
+#include "json.h"
+#include "json_document.h"
+#include "util/pointer.h"
+
+#include "logging.h"
+
+bool ParseAcquireReply(const CurlBuffer& buffer, std::string* session_token) {
+  UniquePtr<JsonDocument> reply(JsonDocument::Create(buffer.data));
+  if (reply->IsValid()) {
+    JSON* result = JsonDocument::SearchInObject(reply->root(),
+                                                "status",
+                                                JSON_STRING);
+    if (result != NULL) {
+      std::string status = result->string_value;
+      if (status == "ok") {
```

Comment:
I think we should use `string` / `char *` constants for the possible return codes.